bin/update_i18n

```diff
@@ -60,8 +60,8 @@ file_put_contents(__DIR__.'/../i18n.php', <<<EOD
  * DO NOT TOUCH THIS FILE!
  *
  * This file is automatically generated by `bin/update_i18n`.
- * Actual Gherkin translations live in cucumber/gherkin repo:
- * {$gherkinUrl}
+ * Actual Gherkin translations live in the cucumber gherkin translation file:
```

Comment:
I would keep the existing message about the fact that the actual translations live in the gherkin repo. I would only replace the URL to point to the github view rather than the raw file. This will help when someone wants to contribute a change.

They will **not** send their translation changes to `__DIR__ . '/../vendor/cucumber/cucumber/gherkin/gherkin-languages.json'`

Reply:
Good point, thanks